### ISSUE TITLE
Crash at the very end of Biker heist

### ISSUE THREAD

**Issue by killrhythm09** [Kamikaze94/WolfHUD#61]
[crash.txt](https://github.com/Kamikaze94/WolfHUD/files/462822/crash.txt)

This happened as soon as we all moved into the escape zone on day 2. The screen went to black, and I crashed. I was host so it killed the entire lobby unfortunately.

**Comment by Kamikaze94** [comment]
Are you on the dev version or release version?
There was a change recently, because someone crashed with the old, because the session seems to got destructed to quickly....
Also, why the heck was there an ECM in your Biker heist...? ^^

**Comment by killrhythm09** [comment]
I was using a dev build that was recent as of this commit:

https://github.com/Kamikaze94/WolfHUD/commit/05cad0e2cf70a17d0501e09741f381470cdd33ed

Also, I remembered some of the details wrong when I posted this initially. Not sure if it makes a difference, but 3 people were in custody and only 1 made it to the escape zone.

Someone brought an ECM jammer to make the turret friendly for a short period so we could move bags without getting fired at constantly. :P
